docs/running.rst

```diff
@@ -317,6 +317,46 @@ There are several recommended ways to run Toil jobs in the cloud. Of these, runn
 
 On all cloud providers, it is recommended that you run long-running jobs on remote systems under ``screen``. Simply type ``screen`` to open a new ``screen` session. Later, type ``ctrl-a`` and then ``d`` to disconnect from it, and run ``screen -r`` to reconnect to it. Commands running under ``screen`` will continue running even when you are disconnected, allowing you to unplug your laptop and take it home without ending your Toil jobs.
 
+.. _Autoscaling:
+
+Autoscaling
+-----------
+The fastest way to get started runnning Toil in a cloud environment is using Toil's autoscaling capabilities to
+handle node provisioning for us. Currently, autoscaling is only supported on the AWS cloud platform with two choices
+of provisioners: CGCloud and Toil's own Docker based provisioner.
+
+The AWS provisioner is included in Toil alongside the [aws] extra and allows us to spin up a cluster without any external dependencies
+using the Toil Appliance docker image. To get started, we choose which version of the appliance to run from CGL's
+`Quay account <https://quay.io/repository/ucsc_cgl/toil-leader?tab=tags>`_ and set the TOIL_APPLIANCE_SELF environment
+variable to quay.io/ucsc-cgl:toil-- followed by the tag of the Toil Appliance version we wish to use::
+
+    export TOIL_APPLIANCE_SELF=quay.io/ucsc_cgl/toil:3.5.0a1--80c340c5204bde016440e78e84350e3c13bd1801
```

Comment:
Can you not hardcode the commit and instead give users a way to find the "latest" image. If not, this is fine, we just need to remember to take this out once #1246 lands.